content/rippleapi_quickstart.md

```diff
@@ -0,0 +1,280 @@
+npm WARN ajv@1.4.10 requires a peer of ajv-i18n@0.1.x but none was installed.
+```
+
+#### 4. (Optional) Tell Git to ignore the `node_modules` folder.
+
+If you are using Git to manage your repository, it's considered a good practice to omit the `node_modules` folder from the Git repo. Other people who check out your code can use `npm` to install the dependencies, and you don't have to keep the repo synchronized with changes to your modules. Edit the `.gitignore` file and add the following line to it:
+
+```
+/node_modules/
+```
+
+Save and commit the changes:
+
+```
+git add .gitignore
+git commit -m "ignore node_modules"
+```
+
+# First RippleAPI Script ##
+
+With RippleAPI installed, it's time to test that it works. Here's a simple script that uses RippleAPI to retrieve information on a specific account:
+
+```
+{% include 'code_samples/rippleapi_quickstart/get-account-info.js' %}
+```
+
+## Running the script ##
+
+RippleAPI and the script both use the ECMAScript 6 version of JavaScript, which is (at this time) not supported by Node.js natively. That's why we installed Babel earlier. The easiest way to run ECMAScript 6 is to use the `babel-node` binary, which NPM installs in the `node_modules/.bin/` directory of your project. Thus, running the script looks like this:
+
+```
+./node_modules/.bin/babel-node get-account-info.js
+```
+
+Output:
+
+```
+getting account info for rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn
+{ sequence: 359,
+  xrpBalance: '75.181663',
+  ownerCount: 4,
+  previousInitiatedTransactionID: 'E5C6DD25B2DCF534056D98A2EFE3B7CFAE4EBC624854DE3FA436F733A56D8BD9',
+  previousAffectingTransactionID: 'E5C6DD25B2DCF534056D98A2EFE3B7CFAE4EBC624854DE3FA436F733A56D8BD9',
+  previousAffectingTransactionLedgerVersion: 18489336 }
+getAccountInfo done
+done and disconnected.
+```
+
+## Understanding the script ##
+
+Even for a simple script, there's a lot packed into that, including some syntax and conventions that are recent developments in JavaScript. Understanding these concepts will help you write better code using RippleAPI, so let's divide the sample code into smaller chunks that are easier to understand.
+
+### Script opening ###
+
+```
+'use strict';
+const {RippleAPI} = require('ripple-lib');
+```
+
+The opening line enables [strict mode](https://www.nczonline.net/blog/2012/03/13/its-time-to-start-using-javascript-strict-mode/). This is purely optional, but it helps you avoid some common pitfalls of JavaScript. See also: [Restrictions on Code in Strict Mode](https://msdn.microsoft.com/library/br230269%28v=vs.94%29.aspx#Anchor_1).
+
+The second line imports RippleAPI into the current scope using Node.js's require function. The [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) assigns it to the variable name `RippleAPI` instead of `ripple-lib` (which is the name of the package, for historical reasons).
```

Comment:
Agree I could drop the whole paragraph, especially if I switch the samples to use the `require('ripple-lib').RippleAPI` syntax we talked about. In its place, it would be nice to link to a summary of other things ripple-lib exports, if such a thing exists.